app/assets/stylesheets/card.scss

```diff
@@ -135,4 +135,69 @@
   .govuk-card__height{
     min-height:220px;
   }
-}
\ No newline at end of file
+}
+
+.new-card {
+  position: relative;
+  background-color: govuk-colour("light-grey");
+  border-bottom: 3px solid govuk-colour("blue");
+
+  &-container {
+    padding: 20px;
+  }
+
+  &:hover, &:focus-within {
+    background-color: govuk-colour("blue");
+    a {
+      color: govuk-colour("white");
+    }
+  }
+
+  &:hover &-link--retake {
+    &:hover {
+      color: #E7F3FD
```

Comment:
Could we give this a variable name or, maybe take `blue` and `lighten` it using that function

Reply:
Done